EIPS/eip-4844.md

```diff
@@ -101,93 +101,28 @@ def fake_exponential(factor: int, numerator: int, denominator: int) -> int:
-In the transaction-pool and local transaction-journal the network encoding is used.
-
-For previous types of transactions the network encoding is no different, i.e. `TransactionNetworkPayload == TransactionPayload`.
-
-The `TransactionNetworkPayload` wraps a `TransactionPayload` with additional data:
-this wrapping data SHOULD be verified directly before or after signature verification.
-
-When a blob transaction is passed through the network (see the [Networking](#networking) section below),
-the `TransactionNetworkPayload` version of the transaction also includes `blobs` and `kzgs` (commitments list).
-The execution layer verifies the wrapper validity against the inner `TransactionPayload` after signature verification as:
-
-- `blob_versioned_hashes` must not be empty
-- All hashes in `blob_versioned_hashes` must start with the byte `BLOB_COMMITMENT_VERSION_KZG`
-- There may be at most `MAX_DATA_GAS_PER_BLOCK // DATA_GAS_PER_BLOB` total blob commitments in a valid block.
-- There is an equal amount of versioned hashes, kzg commitments and blobs.
-- The KZG commitments hash to the versioned hashes, i.e. `kzg_to_versioned_hash(kzg[i]) == versioned_hash[i]`
-- The KZG commitments match the blob contents. (Note: this can be optimized with additional data, using a proof for a
-  random evaluation at two points derived from the commitment and blob data)
-
+The `max_priority_fee_per_gas` and `max_fee_per_gas` fields follow [EIP-1559](./eip-1559.md) semantics
+and `access_list` follows [EIP-2930](./eip-2930.md).
 
-The signature is verified and `tx.origin` is calculated as follows:
+The `blob_versioned_hashes` field represents a list hash outputs from `kzg_to_versioned_hash`.
 
-```python
-def unsigned_tx_hash(tx: SignedBlobTransaction) -> Bytes32:
-    # The pre-image is prefixed with the transaction-type to avoid hash collisions with other tx hashers and types
-    return keccak256(BLOB_TX_TYPE + ssz.serialize(tx.message))
-
-def get_origin(tx: SignedBlobTransaction) -> Address:
-    sig = tx.signature
-    # v = int(y_parity) + 27, same as EIP-1559
-    return ecrecover(unsigned_tx_hash(tx), int(sig.y_parity)+27, sig.r, sig.s)
-```
+The [EIP-2718](./eip-2718.md) `ReceiptPayload` for this transaction is `rlp([status, cumulative_transaction_gas_used, logs_bloom, logs])`.
 
-The hash of a signed blob transaction should be computed as:
+#### Signature
 
-```python
-def signed_tx_hash(tx: SignedBlobTransaction) -> Bytes32:
-    return keccak256(BLOB_TX_TYPE + ssz.serialize(tx))
-```
+The signature values `y_parity`, `r`, and `s` are calculated by constructing a secp256k1 signature over the following digest:
 
-Blob transactions with empty `blob_versioned_hashes` are also considered invalid during execution block verification, and must not be included in a valid block.
+`keccak256(BLOB_TX_TYPE || rlp([chain_id, nonce, max_priority_fee_per_gas, max_fee_per_gas, gas_limit, destination, amount, data, blob_versioned_hashes, access_list]))`.
```

Comment:
Any reason why `max_fee_per_data_gas` and `blob_versioned_hashes` are not appended at the end?

Reply:
Really no reason to not append at end, not sure why I didn't add it originally there. Also was missing the the `max_fee_per_data_gas` field, so I added that back.